resources/views/site/blocks/collection_block.blade.php

```diff
@@ -20,71 +20,79 @@
+                            @slot('variation', 'm-collection-artwork__item')
+                            @slot('item', $item)
+                            @slot('imageSettings', array(
+                                'fit' => null,
+                                'ratio' => null,
+                                'srcset' => ImageHelpers::aic_getSrcsetForImage($item->imageFront(), $item->is_public_domain ?? false),
+                                'sizes' => ImageHelpers::aic_gridListingImageSizes($sizes),
+                            ))
+                            @slot('gtmAttributes', GtmHelpers::combineGtmAttributes([
+                                GtmHelpers::getGtmAttributesForClickMetaDataEventOnArtwork($item),
+                                isset($gtmAttributesForItem)
+                                    ? $gtmAttributesForItem($item, $loop)
+                                    : null,
+                            ]))
+                        @endcomponent
+                    @endif
+                @endforeach
             </div>
-            <div class="collection-btns">
-                <a href="{{ $primary_button_link }}" class="btn f-buttons">{!! $primary_button_label !!}</a>
-                <a href="{{ $secondary_button_link }}" class="btn f-buttons btn--secondary">{!! $secondary_button_label !!}</a>
+            <div class="m-collection-row">
+                @foreach ($artworks as $item)
+                    @if ($loop->even)
+                        @component('components.molecules._m-collection-block--artwork')
+                            @slot('variation', 'm-collection-artwork__item')
+                            @slot('item', $item)
+                            @slot('imageSettings', array(
+                                'fit' => null,
+                                'ratio' => null,
+                                'srcset' => ImageHelpers::aic_getSrcsetForImage($item->imageFront(), $item->is_public_domain ?? false),
+                                'sizes' => ImageHelpers::aic_gridListingImageSizes($sizes),
+                            ))
+                            @slot('gtmAttributes', GtmHelpers::combineGtmAttributes([
+                                GtmHelpers::getGtmAttributesForClickMetaDataEventOnArtwork($item),
+                                isset($gtmAttributesForItem)
+                                    ? $gtmAttributesForItem($item, $loop)
+                                    : null,
+                            ]))
+                        @endcomponent
+                    @endif
+                @endforeach
             </div>
-        @endif
+        </div>
+        <div class="m-collection-bottom">
+            @if($bottom_desc)
+                <div class="collection-desc">
+                    {!! $bottom_desc !!}
+                </div>
+            @endif
+            @if($primary_button_link && $primary_button_label || $secondary_button_link && $secondary_button_label)
```

Comment:
Put this condition around each button, in case they only enter content for one of the buttons.

Reply:
@nikhiltri this is around each button. I think you might've been looking at the stories block branch which doesn't have the conditional rendering changes in it. See below this line in the file

`<div class="collection-btns">
                    @if($primary_button_link && $primary_button_label)
                        <a href="{{ $primary_button_link }}" class="btn f-buttons">{!! $primary_button_label !!}</a>
                    @endif
                    @if($secondary_button_link && $secondary_button_label)
                        <a href="{{ $secondary_button_link }}" class="btn f-buttons btn--secondary">{!! $secondary_button_label !!}</a>
                    @endif
                </div>`